src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -263,4 +269,87 @@ public void verifyEmptyTagsAreNotAppearing() {
         .open(PageContent.PORTABLE_INFOBOX_EMPTY_TAGS)
         .verifyEmptyTags();
   }
+
+  @Test(groups = "PortableInfobox_016")
+  @Execute(onWikia = "mediawiki119")
+  public void insertEmptyInfoboxInVE() {
+    ArticleContent.clear();
+
+    new ArticlePageObject(driver)
+        .open()
+        .openVEModeWithMainEditButton()
+        .clickInsertToolButton()
+        .clickInsertInfoboxFromInsertToolMenu()
+        .selectInfoboxTemplate(2)
+        .clickButtonContaining("Apply changes")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Apply changes" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)